herbert_core/admin/is_idaaas.m

```diff
@@ -21,8 +21,10 @@
     end
     comp_name = getComputerName();
 end
-name_template = 'host_192_168_243';
-if strncmpi(comp_name,name_template,numel(name_template))
+name_templates = {'host_192_168_243','host_172_16'};
```

Comment:
Is this user/host specific?

Reply:
Its IDAaaS  IP-address subspace reflected in host names. Was not able to identify better ways of identifying if the linux machine belongs to iDAaaS cluster